internshipexpectations/overview.md

```diff
@@ -0,0 +1,24 @@
+# Internship Expectations
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=19b692d8-ddf3-4c4f-9f69-ace200043d14&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
+## Takeaways
+
+Takeaways: 
+- The first few weeks are about learning the ropes, you might not be writing code right away and that’s ok!
+- Proactively keep the lines of communication open with your manager and mentor, discuss your learning goals and set specific objectives to work toward. 
+- Internship will make you a better developer, interviewing is a different skill set and takes practice and time! 
+- You can start assessing and interviewing for jobs before you know if you will be converted at your internship. 
+- Internship timeline 
+  * March: Focus on getting to know your team and tools (don't worry about whiteboarding)
+  * April: Focus on mastering basic data structures, taking on new things at internship (don't worry about Midpoint Review)
+  * May: Focus on hitting your stride at internship, mastering more advanced data structures (don't worry about applying to jobs)
+  * June: Focus on applying and interviewing (it's okay to ask your manager to take time off to interview)
+  * July: Focus on getting an offer you're happy with (it's okay if you don't get one from your internship company)
```

Comment:
We might not want these to list specific months since that means we have to update it every cohort:

```suggestion
  * Month 1: Focus on getting to know your team and tools (don't worry about whiteboarding)
  * Month 2: Focus on mastering basic data structures, taking on new things at internship (don't worry about Midpoint Review)
  * Month 3: Focus on hitting your stride at internship, mastering more advanced data structures (don't worry about applying to jobs)
  * Month 4: Focus on applying and interviewing (it's okay to ask your manager to take time off to interview)
  * Month 5: Focus on getting an offer you're happy with (it's okay if you don't get one from your internship company)
```

It's not a big deal, it's just something we will need to make a note of every time.